examples/randomized-benchmark/two-qubit-rb/example_2qb_rb.py

```diff
@@ -0,0 +1,232 @@
+"""
+Define here your Resolver methods such that 
+qubit names "q0" and "q1" are mapped to the correct quantum elements in input config associated 
+to the two qubits you want to benchmark.
+The Resolver allows you to retrieve easily the names of the quantum elements associated to your target qubits, couplers,
+and other elements you might need to carry (fluxlines, etc...).
+
+In this example, we have the simplest case where "q0" and "q1" are defined as the two qubits directly in the config,
+and there is one single coupler named "coupler". Readout resonators coupled to each qubit are named "rr1" and "rr2"
+"""
+
+
+class RBTwoQubitResolver(Resolver):
+    def __init__(self, aliases: Union[dict, None] = None):
+        super().__init__(aliases)
+
+    def q(self, qubit, channel=None):
+        if qubit == "q0":
+            """
+            Put here all commands related to the targeted channel (eg. using two different elements for addressing the 
+            qubit with two distinct frequencies, this can be specified through the channel argument, or any additional 
+            that might be required)
+            """
+            return qubit
+
+        elif qubit == "q1":
+            return qubit
+        else:
+            return qubit
+
+    def res(self, resonator):
+        if resonator == "rr1":
+            return resonator
+        elif resonator == "rr2":
+            return resonator
+
+    def coupler(self, qubit1, qubit2):
+        if qubit1 == 'q0' and qubit1 == 'q1':
+            return "coupler" # Return name of coupling element for your set of qubits
+        else:
+            return coupler
+
+    @property
+    def aliases(self):
+        return self._aliases
+
+
+# Baking Macros required for two qubit gates
+
+# Here is my assumed native two qubit gate
+
+
+resolve = RBTwoQubitResolver()
+
+
+def CZ(b_seq: Baking, ctrl: str = "q0", tgt: str = "q1"):
+    """
+    The native two-qubit gate shall always contain align before and after the playing statements
+    """
+    b_seq.align(resolve.q(ctrl), resolve.q(tgt), resolve.coupler(ctrl, tgt))
+
+    b_seq.play("CZ", resolve.coupler(ctrl, tgt))
+
+    b_seq.align(resolve.q(ctrl), resolve.q(tgt), resolve.coupler(ctrl, tgt))
+
+
+def CNOT(b_seq: Baking, ctrl: str = "q0", tgt: str = "q1"):
+    # Map your pulse sequence for performing a CNOT using baking play statements
+    #
+
+    # Option 1: simple play statement for single qubit gate
+
+    mY_2(b_seq, tgt)
+    CZ(b_seq, ctrl, tgt)
+    Y_2(b_seq, tgt)
+
+
+def iSWAP(b_seq: Baking, ctrl: str = "q0", tgt: str = "q1"):
+    mX_2(b_seq, tgt)
+    CNOT(b_seq, tgt, ctrl)
+    mX_2(b_seq, tgt)
+    CZ(b_seq, ctrl, tgt)
+    Y_2(b_seq, ctrl)
+    X_2(b_seq, tgt)
+
+
+def SWAP(b_seq: Baking, ctrl: str = "q0", tgt: str = "q1"):
+    CNOT(b_seq, ctrl, tgt)
+    CNOT(b_seq, tgt, ctrl)
+    CNOT(b_seq, ctrl, tgt)
+
+
+"""
+In  what follows, q_tgt should be the main target qubit for which should be played the single qubit gate.
```

Comment:
not sure I understand this comment

Reply:
What about "In  what follows, q_tgt is the name of the target qubit on which the gate is applied."
It just means it shall indicate the name of the "q0" or "q1" that you put as argument in the resolver through resolve.q("q0", ...). The name of this argument should be specified. Overall, we need a true documentation for the Resolver use case I figure.